bigtop-deploy/vm/vagrant-puppet-vm/README.md

```diff
@@ -56,13 +56,18 @@ num_instances: 5
 
 5) Test on local built packages is available by:
 
-first, build up local yum repo
```

Comment:
Thanks!  Done -- I removed the README changes and filed a Jira issue for the version bump, BIGTOP-2538.